db/index.go

```diff
@@ -47,11 +47,16 @@ type Index struct {
 
 // GetIndexes returns the current indexes.
 func (c *Collection) GetIndexes() []Index {
-	indexes := make([]Index, len(c.indexes))
+	if len(c.indexes) == 0 {
+		return nil
+	}
+	indexes := make([]Index, len(c.indexes)-1)
 	var i int
 	for _, index := range c.indexes {
-		indexes[i] = index
-		i++
+		if index.Path != idFieldName {
```

Comment:
If we include the index on `_id`, then it has to be filtered by the user if they're going to use these indexes on a _new_ collection, i.e., you can't create a custom index on that field.

Reply:
Ah good 'catch'